bindings/guile/test/CMakeLists.txt

```diff
@@ -41,18 +41,6 @@ gnc_add_scheme_test_targets(scm-test-engine-extras
     OUTPUT_DIR "tests"
     DEPENDS "${GUILE_DEPENDS}")
 
-# Module interfaces deprecated in 4.x, will be removed for 5.x
-gnc_add_scheme_deprecated_module (
-    OLD_MODULE "gnucash engine test test-extras"
-    NEW_MODULE "tests test-engine-extras"
-    DEPENDS "scm-test-engine-extras")
-
-gnc_add_scheme_test_targets(scm-test-engine
```

Comment:
I believe this was removed in error. It's still needed for running tests. Unless you really wanted to remove test scm-test-engine as well ?